plugins/chat/app/serializers/chat/chatables_serializer.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Chat
+  class ChatablesSerializer < ::ApplicationSerializer
+    attributes :users
+    attributes :direct_message_channels
+    attributes :category_channels
+
+    def users
+      (object.users || [])
+        .map do |user|
+          {
+            identifier: "u-#{user.id}",
+            model: ::Chat::ChatableUserSerializer.new(user, scope: scope, root: false),
+            type: "user",
+          }
+        end
+        .as_json
+    end
+
+    def direct_message_channels
```

Comment:
I have to write a serializer spec for this, or maybe move this somewhere else, could maybe do it in the service.

The idea is that I don't want to have duplicates

query: bob
- user: bob
- channel: {users: [bob]}